src/org/labkey/test/tests/ChartingAPITest.java

```diff
@@ -91,6 +93,14 @@ public static void doSetup() throws Exception
     public void initProject()
     {
         _containerHelper.createProject(getProjectName(), null);
+        if (TestProperties.isIgnoreMissingModules())
```

Comment:
Doesn't this need to be if (!TestProperties.isIgnoreMissingModules())?
As it is if TestProperties.isIgnoreMissingModules is true the test will check if the module is there and then throw if it is not, should it check for the module's presence if isIgnoreMissingModules is false?

I'm not familiar with this test but given the class name I'm not sure why this test would want to continue if chartingapi module is missing, so I would expect it to always check for this module regardless of the property setting.

Reply:
No, we want this check to happen if we are ignoring missingModules- the objective is to throw assumptionviolated if the module isn't there (which has the effect of muting the test).

or we could just have it look like this: 
 Set<String> installedModules = _containerHelper.getActiveModules(getProjectName());
            if (TestProperties.isIgnoreMissingModules() && !installedModules.contains("chartingapi") )
            {
                throw new AssumptionViolatedException("Module not installed [chartingapi].  Skipping test.");
            }